### DISCUSSION TITLE
Good repos to contribute to

### DISCUSSION THREAD

**Discussion by Akshu-on-github** [EddieHubCommunity/support#549] [Poll]
I've often heard the question, "What are some good repos you'd suggest contributing to?"

If the ones asking aren't a part of EddieHub, I'd direct them here. But since many are newcomers to the organization, it seems a bit redundant. That's why I figured I'd make a discussion for it.

If anyone finds any good repos, drop a link?

**Comment by emmysteven** [comment]
For those learning .NET Core C#
I'll recommend the following repositories, they all have **good-first-issue**

https://github.com/ardalis/Result
https://github.com/ardalis/CleanArchitecture
https://github.com/ardalis/SmartEnum
https://github.com/ardalis/Specification
https://github.com/ardalis/GuardClauses

**Comment by eddiejaoude** [comment]
Great idea 👍 . As I come across inclusive repos, I will add them to this thread 🤓

**Comment by Vyvy-vi** [comment]
I think linking this here is relevant:
https://github.com/MunGell/awesome-for-beginners

This is a compilation of good beginner-friendly big projects, sorted by language and labels to look for.
